parser/parser.go

```diff
@@ -128,6 +135,7 @@ func Parsers() []string {
 		EDN,
 		VCL,
 		XML,
+		GITIGNORE,
```

Comment:
I wonder if this should be GITIGNORE and DOCKERIGNORE, or just a single parser type of IGNORE.
Either way, it needs fixing.